tests/golden/imagepullsecrets/docker-compose-prod.yml

```diff
@@ -0,0 +1,76 @@
+version: '3.4'
+services:
+  regular-deployment:
+    image: nginx
+    labels:
+      k8ify.imagePullSecret: '$REGULAR_DEPLOYMENT_IMAGEPULLSECRET'
+    environment:
+      - FOO
+      - BAR=${BAR}
+      - something_else=${BAZ}
+      - "PASSWORD=$_ref_:mongodb-secret:password"
+      - "FOOREF=$_ref_:foo:fooooooo"
+      - "BARREF=$_ref_:bar:baaaaaar"
+  
+  regular-statefullset:
```

Comment:
Typo: should be "statefulset" (with a single "l"). Needs to be fixed in many other places too.